src/plugins/auctionHouseModule/AuctionsBuildersClient.ts

```diff
@@ -26,4 +27,8 @@ export class AuctionsBuildersClient {
   createListing(input: CreateListingBuilderParams) {
     return createListingBuilder(this.metaplex, input);
   }
+
+  createBid(input: CreateBidBuilderParams) {
```

Comment:
I know it's a bit picky but since there are going to be a lot of methods for these clients, I'm trying to keep them ordered in alphabetical order. That way, we don't know to think too much about the logical order of these methods when we have 12 of them. Would you mind doing the same for the `AuctionHouseClient`?